NumbersLibrary/RationalNumber.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+
+namespace NumbersLibrary
+{
+    //Дробные числа или Рациональные числа
+    public class RationalNumber
```

Comment:
Надо делать не классом, а структурой.
```suggestion
    public readonly struct RationalNumber
```